packages/ketcher-polymer-editor-react/src/Editor.tsx

```diff
@@ -93,7 +103,20 @@ function Editor() {
           <MenuComponent />
         </Layout.Left>
 
-        <Layout.Main></Layout.Main>
+        <Layout.Main>
+          <svg
+            id="polymer-editor-canvas"
+            ref={canvasRef}
+            width="100%"
+            height="100%"
+          >
+            <defs>
+              <symbol id="peptide" viewBox="0 0 70 61" width="70" height="61">
+                <path d="M16.9236 1.00466C17.2801 0.383231 17.9418 6.10888e-07 18.6583 5.98224e-07L51.3417 2.04752e-08C52.0582 7.81036e-09 52.7199 0.383234 53.0764 1.00466L69.4289 29.5047C69.7826 30.1211 69.7826 30.8789 69.4289 31.4953L53.0764 59.9953C52.7199 60.6168 52.0582 61 51.3417 61H18.6583C17.9418 61 17.2801 60.6168 16.9236 59.9953L0.571095 31.4953C0.217407 30.8789 0.217408 30.1211 0.571096 29.5047L16.9236 1.00466Z"></path>
```

Comment:
Idk, maybe move it to separate file? I looks not very beautiful here xD

Reply:
Yeah good idea. Let me do it in the nest pr related to this topic